src/reference/asciidoc/dsl.adoc

```diff
@@ -491,6 +491,34 @@ public IntegrationFlow loggingFlow() {
 }
 ----
 
+[IMPORTANT]
+=====
+The `log()` or `wireTap()` opearators are applied to the current `MessageChannel` (if it is an instane of `ChannelInterceptorAware`) or an intermediate `DirectChannel` is injected into the flow for the currently configured endpoint.
+In the example above the `WireTap` interceptor is added to the `myChannel` directly, because `DirectChannel` implements `ChannelInterceptorAware`:
```

Comment:
`s/DirectChannel/QueueChannel/`

Reply:
Why? `myChannel()` is a `DirectChannel`, not `QueueChannel`.

Maybe s/above/below? 😄 

Sorry for my bad English

Reply:
Fixed via: https://github.com/spring-projects/spring-integration/commit/2f450f7cde1df2c1596e4a00664e01295f317f3e

Reply:
hehe